src/dart/parse.rs

```diff
@@ -45,6 +52,7 @@ error_chain! {
 }
 
 #[derive(Debug)]
+/// Possible parsing error list.
```

Comment:
"parse" instead of "parsing". Although, not ideal, because this is specifically for "expected X" type of errors.
Maybe "Possible unmet expectations during parsing."?